odoo/addons/base/views/res_partner_views.xml

```diff
@@ -57,7 +57,7 @@
             <field eval="8" name="priority"/>
             <field name="arch" type="xml">
                 <tree string="Contacts" sample="1" multi_edit="1">
-                    <field name="display_name" string="Name"/>
+                    <field name="complete_name" string="Name"/>
```

Comment:
you should not remove the previous field, in this case (res.partner tree view) there is a very high change that it's used in XPATH in customization and that removing it would cause error:

```suggestion
                    <field name="display_name" string="Name" invisible="1"/>
                    <field name="complete_name" string="Name"/>
```

Reply:
thank you, indeed it is used in an XPATH